components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -193,7 +195,7 @@ export const load = (): NewTab.State => {
 export const debouncedSave = debounce<NewTab.State>((data: NewTab.State) => {
   if (data) {
     const dataToSave = {
-      showEmptyPage: data.showEmptyPage,
+      userRegion: data.userRegion,
```

Comment:
It doesn't seem like we need to save this property since it's being fetched fresh every page load.